### PR TITLE
Kimsanboyni ammalab qilgan pull requesti

### PR DESCRIPTION
**Pull request by SharipovIskandar**

### CODE REVIEW

**Comment by SharipovIskandar** [comment]
nimadurda

File: controllers/AdminAdController.php [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    public function delete(int $id): void
+    {
+        $this->ads->deleteAds($id);
+        redirect('/admin/ads');
+    }
+
+    public function search(): void
+    {
+
+        $searchPhrase = $_REQUEST['search_phrase'];
```
Comment (rustam-swe) [inline controllers/AdminAdController.php:116]: Stick to the style or design. Expected to use $_GET for all, but got $_REQUEST

Comment (rustam-swe) [reply controllers/AdminAdController.php:116]: Of course, it could be not your code, just a little tip

File: controllers/home.php [hunk 2]
```diff
@@ -3,5 +3,6 @@
 declare(strict_types=1);
 
 $ads = (new \App\Ads())->getAds();
+$branches = (new \App\Branch())->getBranches();
 
-loadView('home', ['ads' => $ads]);
\ No newline at end of file
+loadView('home', ['ads' => $ads, 'branches' => $branches]);
```
Comment (rustam-swe) [inline controllers/home.php:8]: Could be refactored like this:
loadView('home', [
'ads'      => (new \App\Ads())->getAds(),
'branches' => (new \App\Branch())->getBranches()
]);

File: public/pages/dashboard/ads.php [hunk 3]
```diff
@@ -295,17 +55,17 @@ class="text-lg hover:text-green-600 font-medium ease-in-out duration-500"><?= $a
 
                                 <ul class="py-6 border-y border-slate-100 dark:border-gray-800 flex items-center list-none">
                                     <li class="flex items-center me-4">
-                                        <i class="uil uil-compress-arrows text-2xl me-2 text-green-600"></i>
+                                        <i class="mdi mdi-arrow-expand-all text-2xl me-2 text-green-600"></i>
                                         <span>8000sqf</span>
```
Comment (rustam-swe) [inline public/pages/dashboard/ads.php:59]: Here we expected branch info

File: public/pages/dashboard/ads.php [hunk 4]
```diff
@@ -295,17 +55,17 @@ class="text-lg hover:text-green-600 font-medium ease-in-out duration-500"><?= $a
...
                                     </li>
 
                                     <li class="flex items-center me-4">
-                                        <i class="uil uil-bed-double text-2xl me-2 text-green-600"></i>
+                                        <i class="mdi mdi-bed text-2xl me-2 text-green-600"></i>
                                         <span>4 Beds</span>
```
Comment (rustam-swe) [inline public/pages/dashboard/ads.php:64]: Here goes gender

File: public/pages/dashboard/ads.php [hunk 5]
```diff
@@ -295,17 +55,17 @@ class="text-lg hover:text-green-600 font-medium ease-in-out duration-500"><?= $a
...
                                     </li>
 
                                     <li class="flex items-center">
-                                        <i class="uil uil-bath text-2xl me-2 text-green-600"></i>
+                                        <i class="mdi mdi-shower text-2xl me-2 text-green-600"></i>
                                         <span>4 Baths</span>
```
Comment (rustam-swe) [inline public/pages/dashboard/ads.php:69]: And here rooms

**Review by rustam-swe — COMMENTED** [review]
Well done! In general - good. But why we need components.zip and user-partials.zip in resources/views folder?

**Pull request closed without merging**